### ISSUE TITLE
安卓6。0以上无效

### ISSUE THREAD

**Issue by csrbzhb** [harvbot/android-usb-permission-issuer#1]
安卓6。0以上版本，对于放置在/system/priv-app目录下的APP会进行系统签名校验 。。所以，安卓6。0以上无效。如何解决？

**Comment by OlegKrymskyi** [comment]
Sorry, but you right, it will work only for Android 4.0

**Comment by csrbzhb** [comment]
那只能修改安卓源码，如果有可能的话，否则只能是无解了。

**Comment by OlegKrymskyi** [comment]
Yes, you should remove this check from the android source code and compile it for your device
